tests/check_astunparse.py

```diff
@@ -0,0 +1,25 @@
+import ast
+
+try:
+    import unittest2 as unittest
+except ImportError:
+    import unittest
+
+import test_code_gen
+
+import astunparse
```

Comment:
Since we don't install astunparse dependency, we probably skip the test if astunparse is not available:

```py
try:
    import astunparse
except ImportError:
    astunparse = None


@unittest.skipUnless(astunparse, 'needs astunparse')
class MyTests(test_code_gen.CodegenTestCase):
    ...
```